docs/water_masking.md

```diff
@@ -0,0 +1,88 @@
+
+OSM data was used to generate the water mask for all areas except Canada, Alaska, and Russia. To extract the relevant water extent data from the OSM database, the following filters were applied:
+
+- wr/natural = water 
+- landuse = reservoir
+- waterway = *
+
+In many cases, waterway features stretch from one bank to the other, so islands within those waterways would not be identified as land. To remove islands from the water mask extent, the following filters were applied to the extracted surface water dataset:
+
+- place = island 
+- place = islet
+
+The resulting list of features was exported as a shapefile, then converted to raster format for inclusion in the reference water mask.
+
+#### ESA WorldCover
+
+In October 2021, the European Space Agency (ESA) released the first version of its global land cover dataset, [WorldCover](https://esa-worldcover.org/en/about/about "esa-worldcover.org/en/about" ){target=_blank}. It uses remote sensing data from the Sentinel-1 and Sentinel-2 missions to generate land cover classes, including water. More information is available from the [ESA WorldCover 2020 website](https://worldcover2020.esa.int/ "worldcover2020.esa.int" ){target=_blank}.
+
+This dataset was used to generate the water masks for Canada, Alaska, and Russia. It includes one class for permanent water bodies. The version 1.0 source rasters were downloaded from the [ESA WorldCover 2020 Downloader site](https://worldcover2020.esa.int/downloader "worldcover2020.esa.int/downloader" ){target=_blank}. They were reclassified so that all areas with a value of 80 (Permanent water bodies) were defined as water, and all other values were considered not water.
+
+## Reference Water Mask
+
+The water mask rasters generated from the OSM and WorldCover datasets were mosaicked together, then tiled to 5° latitude by 5° longitude for storage. Because source data is only available from 85°S to 85°N, tiles were added to fill the polar areas. All pixels north of 85°N are treated as water, and all pixels south of 85°S are treated as land.
+
+This reference dataset is stored in a public AWS S3 bucket: 
+
+`s3://asf-dem-west/WATER_MASK/TILES/`
+
+In the reference raster dataset hosted in AWS, pixels with surface water are assigned a value of 1, and all other pixels are assigned a value of 0. 
+
+***Note that the pixel values used in the reference water mask are opposite to the pixel values used for the water masks included in the InSAR product packages. Refer to the [Applying the Water Mask section](#applying-the-water-mask-during-insar-processing "Jump to Applying the Water Mask during InSAR Processing section") for more information on how the reference water mask is transformed during InSAR processing.***
```

Comment:
I forgot what the triple-asterisk syntax (`***`) does, but I assume it renders as you'd expect?